docs/howto/build-run.md

```diff
@@ -149,6 +151,12 @@ Apart from the steps mentioned below, you may find the
 
 [React Native troubleshooting docs]: https://reactnative.dev/docs/troubleshooting
 
+### `react-native run-{android/ios}` launches the app, then crashes immediately
+
+Starting from React Native 0.60, you need to launch the Metro bundler server
+separately, using `react-native start`. Once the server starts up, run
+`react-native run-android` again, and the app should not crash.
+
```

Comment:
A troubleshooting entry would definitely be helpful!

One key element in troubleshooting entries is generally a copy of the error message. That's helpful because it means people can Ctrl+F search for the message they're seeing, and quickly find the entry relevant to them. So let's include a copy of that here.

Reply:
Updated with the error message.